geozarr-spec.md

```diff
@@ -85,44 +85,167 @@ All other CF conventions are recommended, in particular the attributes below:
+A GeoZarr Dataset variable might includes multiscales for a set of DataArray variables.  Also known as "overviews", multiscales provides resampled copies of the original data at a coarser resolution. Multiscales of the original data thus always hold less detail. Common use cases for multiscales are fast rendering for visualization purposes and analyzing data at multiple resolution. 
 
 ### Multiscales Encoding 
 
- Multiscales MUST be encoded in children groups.
+ Multiscales MUST be encoded in children groups. Data at all scales MUST use the same coordinate reference system and must follow ONE common zoom level strategy. The zoom level strategy is modelled in close alignment to the [OGC Two Dimensional Tile Matrix Set](https://docs.ogc.org/is/17-083r4/17-083r4.html) version 2 and the [Tiled Asset STAC extension](https://github.com/stac-extensions/tiled-assets). Each zoom level is described by a Matrix defining the number, layout, origin and pixel size of included tiles. These tiles MUST correspond to the chunk layout along the two spatial dimensions listed in `_ARRAY_DIMENSIONS` of a given group.
  
-* Multiscale group name is the zoom level (e.g. '0').
-* Multiscale group contains all DataArrays generated for this specific zoom level.
-* Zoom level  strategy is based on defacto standard level 0 as 256x256 pixels covering the entire world, and scale doubled on each level as per https://wiki.openstreetmap.org/wiki/Zoom_levels. 
-* Multiscale chunking is RECOMMENDED to be 256 pixels or 512 pixels for the latittude and longitude dimensions.
+* Multiscale group name is the zoom level identifier (e.g. '0').
+* Multiscale group contains all DataArrays generated for this specific zoom level. 
+* Multiscale chunking is RECOMMENDED to be 256 pixels or 512 pixels for the two spatial dimensions listed in `_ARRAY_DIMENSIONS`.
 
 ### Multiscales Metadata
 
-Each DataArray MUST define the 'multiscales' metadata attribute that provides the multiscales group path :
+If implemented, each DataArray MUST define the 'multiscales' metadata attribute which includes the following fields:
+* `tile_matrix_set`
+* `tile_matrix_set_limits` (optional)
+* `resampling_method`
+
+
+#### Tile Matrix Set
+Tile Matrix Set can be: 
+* the name of a well know tile matrix set. Well known Tile Matrix Sets are listed [here](https://schemas.opengis.net/tms/2.0/json/examples/tilematrixset/).
+* the URI of a JSON document describing the Tile Matrix Set following the OGC standard.
+* a JSON object describing the Tile Matrix Set following the OGC standard (CamelCase!).
+
+Within the Tile Matrix Set
+* the Tile Matrix identifier for each zoom level MUST be the relative path to the Zarr group which holds the DataArray variable 
+* zoom levels MUST be provided from lowest to highest resolutions
+* the `supportedCRS` attribute of the Tile Matrix Set MUST match the crs information defined under **grid_mapping**.
```

Comment:
Do we need the duplication?

Reply:
There is no `supportedCRS` attribute in [2DTMS 2.0](https://docs.ogc.org/is/17-083r4/17-083r4.html#toc16).

See also the [Release Notes](https://docs.ogc.org/is/17-083r4/21-066r1.html#_renaming_supportedcrs_to_crs) explaining how `supportedCRS` was changed to `crs`, as well as [6.2.1.1.  TileMatrixSet CRS Compatibility](https://docs.ogc.org/is/17-083r4/17-083r4.html#toc16) explaining that a 2DTMS is compatible with multiple CRS (e.g., CRS that only change axis order -- as that does not affect the tiling at all, CRSs that add additional dimension (e.g. CRS84 vs CRS84h), as well as realizations of a datum ensemble .